src/coreclr/inc/eetwain.h

```diff
@@ -396,7 +403,6 @@ TADDR GetAmbientSP(PREGDISPLAY     pContext,
     Get the number of bytes used for stack parameters.
     This is currently only used on x86.
 */
-virtual
```

Comment:
Is there a pateren for why `virtual` was removed on some methods and not on the others?

For example, why `GetAmbientSP` a few lines above has virtual, but `GetStackParameterSize` lost it?

Reply:
That was actually not intentional. The way it happened was that I've first converted all methods to use "override" instead of just "virtual", but the compilation on Linux started to fail because the compiler when it saw override somewhere wanted it everywhere and not just on the classes I had it on, but it wanted it on the base class and I believe also on some related classes. So I've reverted that to not to expand the change by unnecessary changes, but missed couple of places were I've removed the virtual.